ci-operator/config/operator-framework/helm/operator-framework-helm-master.yaml

```diff
@@ -8,7 +8,7 @@ base_images:
   base:
     name: '4.0'
     namespace: ocp
-    tag: base
+    tag: centos
```

Comment:
We shouldn’t be tagging centos into this (if it’s there I’m going to delete it).

Use namespace: openshift, name centos, tag 7

Reply:
Ok